tests/data/fixtures/pyrflx.py

```diff
@@ -203,3 +204,15 @@ def fixture_message_type_size_value(
     message_type_size_package: pyrflx.Package,
 ) -> pyrflx.MessageValue:
     return message_type_size_package["Message"]
+
+
+@pytest.fixture(name="always_valid_aspect_package", scope="session")
+def fixture_always_valid_aspect(pyrflx_: pyrflx.PyRFLX) -> pyrflx.Package:
+    return pyrflx_["Always_Valid_Aspect"]
+
+
+@pytest.fixture(name="message_always_valid_aspect")
+def fixture_always_valid_aspect_value(
```

Comment:
`message_always_valid_aspect_value` is rejected as an argument name for `test_always_valid_aspect` by pylint because it is too long, Because of that I used a shorter form for it. Should I deactivate the `invalid-name` at this line?

Reply:
I would favor disabling the pylint check for this line.

Reply:
Fixed it in  707f5c8